.vscode/settings.json

```diff
@@ -1,9 +1,14 @@
 {
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#4459ff",
-    "titleBar.activeForeground": "#fff",
-    "titleBar.inactiveBackground": "#4459ff1a",
-    "titleBar.inactiveForeground": "#858b9a",
-    "activityBar.activeBorder": "#4459ff"
-  }
+  "tailwindCSS.experimental.configFile": {
+    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
+    "packages/design-system-twrnc-preset/tailwind.config.js": "packages/design-system-react-native/src/**"
+  },
```

Comment:
This setting allows us to manually specify the Tailwind configuration file as we don't use them directly in the component library packages

https://github.com/tailwindlabs/tailwindcss-intellisense?tab=readme-ov-file#tailwindcssexperimentalconfigfile